docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -0,0 +1,217 @@
+---
+sidebar_label: ClickPipes (New)
+description: Seamlessly connect your external data sources to ClickHouse Cloud. 
+slug: /en/integrations/clickpipes
+---
+
+import KafkaSVG from "../../images/logos/kafka.svg";
+import ConfluentSVG from "../../images/logos/confluent.svg";
+
+
+# Integrating Kafka with ClickHouse Cloud
+
+## Introduction
+
+[ClickPipes](https://clickhouse.com/cloud/clickpipes) is a managed integration platform that makes ingesting data from a diverse set of sources as simple as clicking a few buttons. Designed for the most demanding workloads, ClickPipes's robust and scalable architecture ensures consistent performance and reliability. 
+<br/>
+
+<img src={require('./images/clickpipes_stack.png').default} class="image" alt="ClickPipes Stack Illustration"/>
+
+<br/>
+
+:::note
+ClickPipes is a native capability of [ClickHouse Cloud](https://clickhouse.com/cloud) currently under private preview. You can join [our waitlist here](https://clickhouse.com/cloud/clickpipes#joinwaitlist)
+:::
+
+
+## Setup
+
+### 1. Enable ClickPipes for your cloud organization
+
+ClickPipes is currently accessible in private preview. You can join our waitlist by filling [this form](https://clickhouse.com/cloud/clickpipes#joinwaitlist). Please note that during the Private Preview phase, ClickPipes is available only for Amazon Web Services backed services, in the `us-east-2` region.
+
+
+### 2. Creating your first ClickPipe
+
+1. Access the SQL Console for your ClickHouse Cloud Service running in AWS `us-east-2` region.
```

Comment:
Should we mention that we recommend creating a new instance? Even if it's mentioned below

Reply:
The mention below should be enough